HW2/PriceCalc.py

```diff
@@ -0,0 +1,6 @@
+Price = float(input('Enter price:'))
```

Comment:
this input allows me to enter fractional kopecks
please rework it to receive only valid price.